FiveCalls/FiveCalls/AppState.swift

```diff
@@ -49,9 +49,26 @@ class AppState: ObservableObject {
     }
 }
 
-protocol AppStateDelegate {
-    func setIssues(issues: [Issue])
-    func setContacts(contacts: [Contact])
-    func setFetchingContacts(fetching: Bool)
-    func setLocation(location: NewUserLocation)
+extension AppState {
+    @MainActor
+    func setLocation(newLocation: NewUserLocation) {
+        location = newLocation
+        
+        Operator().fetchContacts(location: newLocation, fetching: self.setFetchingContacts, setContacts: self.setContacts)
```

Comment:
So we wouldn't want to use `MainActor` for this whole method since I think it would then run the operation on the main thread, which we don't want.

Reply:
I figured we could indicate the whole `AppState` class as `MainActor` and wouldn't have to repeat it for each function below, however, for this one or at least the fetch, we'd want to disqualify.

Reply:
I guess I was continuing the assumption that the operation execution runs on some background thread regardless of the context it was called on. I don't actually know how this interacts with `@MainActor`!